scripts/knot_benchmark/README.md

```diff
@@ -0,0 +1,15 @@
+## Getting started
+
+1. [Install `uv`](https://docs.astral.sh/uv/getting-started/installation/)
+
+- Unix: `curl -LsSf https://astral.sh/uv/install.sh | sh`
+- Windows: `powershell -c "irm https://astral.sh/uv/install.ps1 | iex"`
+
+1. Build red_knot: `cargo build --bin red_knot --release`
+1. Run benchmarks: `uv run benchmark`
```

Comment:
```
~/dev/ruff (knot-mypy-pyright-benchmarks) % uv run benchmark                  
   Built ruff @ file:///Users/alexw/dev/ruff
Installed 1 package in 1ms
error: Failed to spawn: `benchmark`
  Caused by: No such file or directory (os error 2)
```

Do I need to `cd` into the `scripts/red_knot_benchmark` directory?

Reply:
Possibly

Reply:
It ([nearly](https://github.com/astral-sh/ruff/pull/13026#discussion_r1724856235)) worked after I did that, so I think the answer is "yes" 😄

```suggestion
1. `cd` into the benchmark directory: `cd scripts/knot_benchmark`
1. Run benchmarks: `uv run benchmark`
```